.github/workflows/main.yml

```diff
@@ -141,23 +142,24 @@ jobs:
       # Perform various setup operations
       # --------------------------------------------      
       - name: Setup - Prepare OS
-        uses: virtualsatellite/ci-actions/ci-setup-action@v2
+        uses: virtualsatellite/ci-actions/ci-setup-action@v6-beta
```

Comment:
I guess we could change that to v6? We have it released already?

Reply:
Nice find! Updated to v6.